src/DssSpell.sol

```diff
@@ -19,76 +19,67 @@ pragma solidity 0.8.16;
+    function restrict(uint256) external;
+}
 interface RwaLiquidationLike {
-    function ilks(bytes32) external view returns (string memory doc, address pip, uint48 tau, uint48 toc);
-    function init(bytes32 ilk, uint256 val, string calldata doc, uint48 tau) external;
+    function ilks(bytes32) external view returns (string memory, address, uint48, uint48);
+    function init(bytes32, uint256, string calldata, uint48) external;
+}
+
+interface ACLManagerLike {
+    function addPoolAdmin(address admin) external;
+}
+
+interface ProxyLike {
+    function exec(address target, bytes calldata args) external payable returns (bytes memory out);
+}
+
+interface Initializable {
+    function init(bytes32 ilk) external;
 }
 
 interface RwaUrnLike {
-    function lock(uint256 wad) external;
     function hope(address usr) external;
+    function nope(address usr) external;
+    function lock(uint256 wad) external;
+    function draw(uint256 wad) external;
 }
 
 interface RwaInputConduitLike {
     function mate(address usr) external;
+    function hate(address usr) external;
     function file(bytes32 what, address data) external;
 }
 
 interface RwaOutputConduitLike {
     function file(bytes32 what, address data) external;
     function hope(address usr) external;
+    function nope(address usr) external;
     function mate(address usr) external;
+    function hate(address usr) external;
     function kiss(address who) external;
-}
-
-interface PoolConfiguratorLike {
-    struct InitReserveInput {
-        address aTokenImpl;
-        address stableDebtTokenImpl;
-        address variableDebtTokenImpl;
-        uint8 underlyingAssetDecimals;
-        address interestRateStrategyAddress;
-        address underlyingAsset;
-        address treasury;
-        address incentivesController;
-        string aTokenName;
-        string aTokenSymbol;
-        string variableDebtTokenName;
-        string variableDebtTokenSymbol;
-        string stableDebtTokenName;
-        string stableDebtTokenSymbol;
-        bytes params;
-    }
-    function initReserves(InitReserveInput[] calldata input) external;
-    function configureReserveAsCollateral(
-        address asset,
-        uint256 ltv,
-        uint256 liquidationThreshold,
-        uint256 liquidationBonus
-    ) external;
-    function setBorrowableInIsolation(address asset, bool borrowable) external;
-    function setDebtCeiling(address asset, uint256 newDebtCeiling) external;
-}
-
-interface AaveOracleLike {
-    function setAssetSources(address[] calldata assets, address[] calldata sources) external;
+    function pick(address who) external;
+    function push() external;
+    function push(uint256 wad) external;
 }
 
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     string public constant override description = "Goerli Spell";
+    VatLike internal immutable vat = VatLike(DssExecLib.getChangelogAddress("MCD_VAT"));
```

Comment:
Move these down to where the other addresses are used (after all constant address declarations for example)